client/blocks/eligibility-warnings/warning-list.tsx

```diff
@@ -121,25 +123,14 @@ function getWarningDescription(
 			);
 
 		case 'hosting-features':
-			return hasTranslation(
-				'By activating all hosting features the following change will be made to the site:'
-			)
-				? translate(
-						'By activating all hosting features the following change will be made to the site:',
-						'By activating all hosting features the following changes will be made to the site:',
-						{
-							count: warningCount,
-							args: warningCount,
-						}
-				  )
-				: translate(
-						'By activating all developer tools the following change will be made to the site:',
-						'By activating all developer tools the following changes will be made to the site:',
-						{
-							count: warningCount,
-							args: warningCount,
-						}
-				  );
+			return translate(
+				'By proceeding the following change will be made to the site:',
+				'By proceeding the following changes will be made to the site:',
```

Comment:
I see we are both getting rid of the old `hasTranslation` check while adding a new translation. I see the `stringFreeze` tag is here so Im assuming the plan is to hold for main translations?

Reply:
@Addison-Stavlo The new copy is somehow already translated: https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/hosting-features-activate-modal

Maybe it was used before?

<img width="722" alt="Screen Shot 2024-09-18 at 11 35 23 AM" src="https://github.com/user-attachments/assets/fdd68b78-614b-4fa6-a422-d19423632878">

Reply:
awesome!